src/group_theory/free_group.lean

```diff
@@ -369,10 +369,23 @@ instance : has_mul (free_group α) :=
     (λ L₁ L₂ H, quot.induction_on y $ λ L₃, quot.sound $ red.step.append_right H)⟩
 @[simp] lemma mul_mk : mk L₁ * mk L₂ = mk (L₁ ++ L₂) := rfl
 
+/-- Transform a word representing a free group element into a word representing its inverse. --/
+def inv_rev (w : list (α × bool)) : list (α × bool) :=
+(list.map (λ (g : α × bool), (g.1, bnot g.2)) w).reverse 
+
+@[simp] lemma inv_rev_eq (w : list (α × bool)) : inv_rev w =  
+(list.map (λ (g : α × bool), (g.1, bnot g.2)) w).reverse := rfl 
```

Comment:
I'm not sure that we want to unfold every instance of `inv_rev`. Usually, if you add a definition, then you add API lemmas about it instead of just unfolding it every time.

Reply:
The relevant lemma `inv_rev_length` is there, and it seems I can use `simp [inv_rev]` to include the unfolding in the couple places in the original code.  So I deleted `inv_rev_eq` in 037eac7c61208ed0a724b079fd23081329cc0de5.